create-service-and-ensure-existence.sh

```diff
@@ -0,0 +1,18 @@
+organization=$1
```

Comment:
Are these parameters mandatory?
if so they should be `${1?}`

Reply:
No. It is optional.